docs/api_requests/block-io-engine.md

```diff
@@ -65,3 +70,82 @@ This translates to higher latencies on either of these operations:
 
 For use-cases where the lowest latency on the aforementioned operations is
 desired, it is recommended to use the `Sync` IO engine.
+
+### Block IOPS and efficiency
+
```

Comment:
Do we also mention the downsides like increased startup time, cpu usage and maybe memory, that might be seen in oversubscription enviroments.

Reply:
We do.
We talk about the increased startup time in the previous paragraph and about the increased CPU consumption in the paragraph about write workloads